pyqir-parser/Cargo.toml

```diff
@@ -24,6 +24,7 @@ name = "pyqir_parser"
 crate-type = ["cdylib"]
 
 [package.metadata.maturin]
+name = "pyqir.parser._native"
```

Comment:
Ah, I see it's part of something maturin related here. Maybe a comment in the rust indicating that `_native` is used this way?

Reply:
Is it the name needing to be the same that's confusing, or specifically the underscore? If it's the underscore, it can alternatively be defined as

```rust
#[pymodule(name = "_native")]
fn native(_py: Python<'_>, m: &PyModule) -> PyResult<()> { ... }
```

Do you prefer that?

Reply:
The name confuses me just because prepending with underscore usually means a rust function or variable unused. I've seen clippy flag any unused variables that aren't prepended and flag any variables prepended that are actually used, so seeing this `_native` made me think it was in fact an unused function. But given that this is one of the places where two conventions from different languages are meeting, I am fine if the underscore is kept. Whatever makes the most sense!

Reply:
I agree that it makes it look like an unused function in Rust. I renamed them to `native_module` with a pyo3 attribute to name them `_native` in the pyd.

Reply:
Great,, thanks!